lib/errorCheck.js

```diff
@@ -0,0 +1,22 @@
+export function isInvalidAccessTokenError({error}) {
+  if(error.status === 401 && error.data &&
+    (error.data.errors && error.data.errors.some(({name}) => name ===
+    'ConstraintError') || error.data.name === 'NotAllowedError' ||
```

Comment:
Let's remove `NotAllowedError`. On further reflection, that's not going to be a recoverable error.